index.js

```diff
@@ -127,11 +156,20 @@ export function openDittoTimely(eventName, params = {}, env = "prod") {
 
   // set content
   modal.setContent(
-    `<iframe id="timely-iframe" style="width: 100%;height:100%;border:none;" src="${timelyUrlWithParams}"></iframe>`
+    `
+    <div class="parent-container">
+    <button id="close-button-timely" class="custom-close-button-timely">
+    <svg xmlns="http://www.w3.org/2000/svg" width="32" height="32" viewBox="0 0 15 15"><path fill="#FFFFFF" fill-rule="evenodd" d="M11.782 4.032a.575.575 0 1 0-.813-.814L7.5 6.687L4.032 3.218a.575.575 0 0 0-.814.814L6.687 7.5l-3.469 3.468a.575.575 0 0 0 .814.814L7.5 8.313l3.469 3.469a.575.575 0 0 0 .813-.814L8.313 7.5l3.469-3.468Z" clip-rule="evenodd"/></svg>
+    </button>
+    <iframe id="timely-iframe" style="width: 100%;height:100%;border:none;" src="${timelyUrlWithParams}"></iframe>
+    <div>
+    `
   );
 
+  const closeButton = window.document.getElementById("close-button-timely");
+  closeButton.addEventListener("click", handleCloseButtonClick);
```

Comment:
better remove the listner whenever modal is closed

Reply:
it is done in line 130

Reply:
cool..missed that.